content/applications/finance/accounting/taxes/avatax/avalara_portal.rst

```diff
@@ -0,0 +1,182 @@
+user's favorites.
+
+.. tip::
+   Optionally, select a pre-configured report from the :guilabel:`Frequently used reports` section
+   of the reporting dashboard. Access reporting by clicking on the :guilabel:`Reports` option in the
+   top menu of the Avalara management console.
+
+Add more jurisdictions
+======================
+
+Additional jurisdictions can be added (tax locations) in the Avalara management console. Navigate to
+either Avalara's `sandbox <https://sandbox.admin.avalara.com/>`_ or `production
+<https://admin.avalara.com/>`_ environment. This will depend on which type of account was set in the
+:doc:`integration <../avatax>`.
+
+Next, navigate to :menuselection:`Settings --> Where you collect taxes`. Choose between the three
+different tabs depending on the business need. The first tab is :guilabel:`Sales and use tax` where
+tax can be collected for the United States. Click the :icon:`fa-plus` :guilabel:`Add to where you
+collect sales and use tax` to add another location where the company collects sales and use tax.
+
+The second option, is the :guilabel:`VAT / GST` tab where the :icon:`fa-plus` :guilabel:`Add a
+country or territory where you collect VAT / GST` can be selected to add another country or
+territory where the company collects VAT/GST.
+
+Finally, on the far-right is the :guilabel:`Customs Duty` tab, where a country can be added where
+the company collects customs duty. Simply click on the :icon:`fa-plus` :guilabel:`Add a country
+where you calculate customs duty` icon below the tab.
+
+.. image:: avalara_portal/where-you-collect-tax.png
+   :align: center
+   :alt: AvaTax management console, on the Where you collect tax page, with the add button and
+         sales and use tax tab highlighted.
+
+Tax exemption certificate
+=========================
+
+Tax exemption certificates for customers can be added into the Avalara management console so that
+*AvaTax* is aware of which customers may be exempt from paying certain taxes. To add an *exception
+certificate* navigate to :menuselection:`Exemptions --> Customer certificates`. From there, click on
+the :icon:`fa-plus` :guilabel:`Add a certificate` to configure an exemption.
+
+.. warning::
+   An Avalara subscription to Exemption Certificate Management (ECM) is required in order to attach
+   certificate images and to be ready for an audit. For more on subscribing to this add-on contact
+   `Avalara <https://www.avalara.com/us/en/index.html>`_.
+
+End-of-year operations
+======================
+
+Avalara's services include tax return services, for when it is time to file taxes at the end of the
+year. To access Avalara's tax services log into the `management portal
+<https://admin.avalara.com/>`_ and on the main dashboard, click :guilabel:`Returns`. Avalara will
+prompt the Avalara user to log in for security purposes and redirect the user to the *Returns*
+portal.
+
+Click :guilabel:`Get started` to begin the tax return process.
```

Comment:
is there an Avalara tax doc you can link ot htese instructions? Just because I saw you linked ot a diff doc earlier in the doc.